packages/edit-navigation/src/components/menu-editor/index.js

```diff
@@ -35,24 +33,35 @@ export default function MenuEditor( { menuId, blockEditorSettings } ) {
 					templateLock: 'all',
 				} }
 			>
-				<div className="edit-navigation-menu-editor__panel">
-					{ !! blocks.length && (
-						<__experimentalBlockNavigationList
-							blocks={ blocks }
-							selectedBlockClientId={ blocks[ 0 ].clientId }
-							selectBlock={ () => {} }
-							showNestedBlocks
-							showAppender
-						/>
-					) }
-				</div>
-				<div className="edit-navigation-menu-editor__panel">
-					<WritingFlow>
-						<ObserveTyping>
-							<BlockList />
-						</ObserveTyping>
-					</WritingFlow>
-				</div>
+				<Panel className="edit-navigation-menu-editor__panel">
+					<PanelBody title={ __( 'Navigation structure' ) }>
+						{ !! blocks.length && (
+							<__experimentalBlockNavigationList
+								blocks={ blocks }
+								selectedBlockClientId={ blocks[ 0 ].clientId }
+								selectBlock={ () => {} }
+								showNestedBlocks
+								showAppender
+							/>
+						) }
+					</PanelBody>
+				</Panel>
+				<Panel
+					header={
+						<Button isPrimary onClick={ saveBlocks }>
+							{ __( 'Save Navigation' ) }
```

Comment:
Let's use sentence casing here (see https://github.com/WordPress/gutenberg/pull/18758).

```suggestion
							{ __( 'Save navigation' ) }
```